app/services/publish_content_item.rb

```diff
@@ -0,0 +1,34 @@
+class PublishContentItem
```

Comment:
We should try to find a name that reflects the "special" behaviour here, which is the looking up of content id by base path before publishing.